src/net/account/account_client.py

```diff
@@ -1,13 +1,29 @@
+import os.path
 from dataclasses import dataclass
 from typing import Dict, Optional, List
 
 from starkware.cairo.common.hash_state import compute_hash_on_elements
 from starkware.starknet.definitions.transaction_type import TransactionType
 from starkware.starknet.public.abi import get_selector_from_name
 from starkware.starknet.services.api.gateway.transaction import InvokeFunction
-from starkware.crypto.signature.signature import private_to_stark_key, sign
+from starkware.crypto.signature.signature import (
+    private_to_stark_key,
+    sign,
+    get_random_private_key,
+)
+
+from src.contract import Contract
 from src.net import Client
-from src.utils.types import AddressRepresentation, parse_address
+from src.utils.types import (
+    AddressRepresentation,
+    parse_address,
+    NetType,
+    net_address_from_type,
+)
+
+directory = os.path.dirname(__file__)
+compiled_account_contract_fname = os.path.join(directory, "Account-compiled.json")
+compiled_account_contract = open(compiled_account_contract_fname).read()
```

Comment:
This is a module's side effect during module initialisation and it will block execution in asynchronous context. I suggest we place this json directly in a python file, we can add script for generating it in the future.